fastlane/Fastfile

```diff
@@ -39,6 +39,34 @@ platform :android do
     attach_changelog_to_main(options)
   end
 
+  desc "Tag release version with latest if necessary"
+  lane :tag_release_with_latest_if_needed do |options|
+    release_version = options[:release_version]
+
+    lines_in_changelog = File.open("../CHANGELOG.md").to_a
+    first_line = lines_in_changelog.first
+    latest_version = first_line.split("## ").last.chomp
+    puts "Latest version is #{latest_version}"
+
+    if latest_version == release_version then
```

Comment:
should this be <=?

Reply:
oh i guess it's not an integer...could split on "." and create an integer if we wanted to be able to compare

Reply:
So if we are releasing 4.3.0 (release_version is 4.3.0), and the latest version in the CHANGELOG is 4.3.0 (it has just been released so it should be there), then that's the latest version.

If we are releasing 4.0.1, and the latest version in the CHANGELOG is 4.3.0, then 4.0.1 is not the latest version.

Reply:
ohhh forgot changelog is already updated at this point, got it